src/helpers deel2/listTvsForSport.js

```diff
@@ -0,0 +1,15 @@
+import {inventory} from "../constants/inventory.js";
+
+function listedTvsSport () {
+    const tvsSport = inventory.map((tvSport) => {
```

Comment:
Uit de map komt een array met alleen maar `undefined`. En vervolgens doe je niets met `tvsSport`. In dat geval kun je beter een forEach gebruiken
```suggestion
    inventory.forEach((tvSport) => {
```